code/modules/random_map/drop/drop_types.dm

```diff
@@ -61,9 +61,9 @@ var/global/list/datum/supply_drop_loot/supply_drop
 		/obj/item/weapon/shield/riot/tele,
 		/obj/item/weapon/storage/box/emps,
 		/obj/item/weapon/storage/box/flashbangs,
-		/obj/item/weapon/gun/projectile/automatic/sts35,
-		/obj/item/ammo_magazine/c762/ap,
-		/obj/item/ammo_magazine/c762/ap,
+		/obj/item/weapon/gun/projectile/automatic/carbine,
```

Comment:
Oh yea, I changed this too, because I didn't want to leave any loose ends where the OP gun could be obtained.